server/spec/owner_resource_spec.rb

```diff
@@ -185,7 +185,7 @@
     #and this will fail
     lambda do
       ro_owner_client.register('systemFoo')
-    end.should raise_exception(RestClient::ResourceNotFound)
+    end.should raise_exception(RestClient::Forbidden)
   end
```

Comment:
Let's add another spec test to cover the different exception for the case where the user has no visibility to the owner.